### PR TITLE
Mark email fields as safe and remove repetition in email header

### PR DESCRIPTION
**Pull request by dragon-dxw**
The text of the email must be interpretted without escaping, as there is no process that
will unescape it, unlike in a browser context.

Addresses https://trello.com/c/baGbbWHc/922-eui-changing-to-the-wording-of-email-from-eui-needed

We also remove a doubling up of the subject name -- Find Case Law is automatically prepended

### CODE REVIEW

**Comment by dragon-dxw** [comment]
[EUI: Changing to the wording of email from EUI needed](https://trello.com/c/baGbbWHc/922-eui-changing-to-the-wording-of-email-from-eui-needed)

File: ds_caselaw_editor_ui/templates/emails/confirmation_to_submitter.txt [hunk 1]
```diff
@@ -1,12 +1,12 @@
-Dear {{ submitter }},
+Dear {{ submitter|safe}},
```
Comment (anthonyhashemi) [inline ds_caselaw_editor_ui/templates/emails/confirmation_to_submitter.txt:1]: can we add tests asserting the correct strings in the rendered template?

Comment (dragon-dxw) [reply ds_caselaw_editor_ui/templates/emails/confirmation_to_submitter.txt]: Added.

**Review by anthonyhashemi — APPROVED** [review]
Nice one 👍

**Pull request merged**